illumos-utils/src/running_zone.rs

```diff
@@ -638,7 +641,7 @@ impl InstalledZone {
 
         let full_zone_name = Self::get_zone_name(zone_type, unique_name);
         let zone_image_path =
-            PathBuf::from(&format!("/opt/oxide/{}.tar.gz", zone_type));
+            Utf8PathBuf::from(&format!("/opt/oxide/{}.tar.gz", zone_type));
```

Comment:
No need to pass a `&String` here:

```suggestion
            Utf8PathBuf::from(format!("/opt/oxide/{}.tar.gz", zone_type));
```

Reply:
Done in https://github.com/oxidecomputer/omicron/pull/2988/commits/ced5dc92aa41c74da6a846aaa9cd59d20d9c7843